src/Util/Notifications.js

```diff
@@ -1,11 +1,11 @@
 import 'react-notifications-component/dist/theme.css'
 import { Store } from 'react-notifications-component'
 
-function displaySucessNotification(title,message){
+function displayNotification(title,message,type){
     Store.addNotification({
         title: title,
         message: message,
-        type: "success",
+        type: type,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9156896)
```suggestion
        type,
```